test/unit/typescript/decorators.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../../../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../../../web-component-tester/browser.js"></script>
+    <!--
+      This test uses IMD for simplicity and less tools, but browserify,
+      webpack, or rollup could probably be used relatively easily to build a
+      single JS bundle
+    -->
+    <script src="../../../../imd/imd.js"></script>
+    <script src="../../../../chai/chai.js" as="chai"></script>
+    <script
+      src="../../../node_modules/reflect-metadata/Reflect.js">
+    </script>
+    <script>
+      define('reflect-metadata', () => {
```

Comment:
I want to try a webpack setup rather than IMD. That might improve things here. As for reflect-metadata, I need to learn a lot more about it...